src/Controls/src/Core/Compatibility/Handlers/Windows/FrameRenderer.cs

```diff
@@ -67,16 +67,7 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 		protected override void UpdateBackgroundColor()
 		{
-			// Background color change must be handled separately
-			// because the background would protrude through the border if the corners are rounded
-			// as the background would be applied to the renderer's FrameworkElement
-			Color backgroundColor = Element.BackgroundColor;
-
-			if (Control != null)
-			{
-				Control.Background = backgroundColor.IsDefault() ?
-					new Microsoft.UI.Xaml.Media.SolidColorBrush((global::Windows.UI.Color)Resources["SystemAltHighColor"]) : backgroundColor.ToPlatform();
-			}
+			UpdateBackground();
```

Comment:
With this changes we render correctly the Background brush (also the background color).